.mergify.yml

```diff
@@ -3,27 +3,14 @@ pull_request_rules:
     conditions:
       - base=master
       - "#approved-reviews-by>=1"
-      - label!=work-in-progress
       - label!=manual merge
       - "check-success=lintSrc (10.x)"
       - "check-success=lintSrc (12.x)"
       - "check-success=lintSrc (14.x)"
     actions:
       merge:
         strict: "smart"
-        method: rebase
-  - name: automatic merge from dependabot
-    conditions:
-      - author=dependabot[bot]
-      - label!=work-in-progress
-      - label!=manual merge
-      - "check-success=lintSrc (10.x)"
-      - "check-success=lintSrc (12.x)"
-      - "check-success=lintSrc (14.x)"
-    actions:
-      merge:
-        strict: "smart"
-        method: rebase
+        method: squash
```

Comment:
Why did you choose squash instead of rebase?

Reply:
Because I expect people to contribute and not work correctly with atomic commits. 😁